superset/assets/javascripts/explore/components/controls/AnnotationLayer.jsx

```diff
@@ -0,0 +1,558 @@
+                onChange={(v) => {
+                  delete overrides.until;
+                  if (v) {
+                    this.setState({ overrides: { ...overrides, until: null } });
+                  } else {
+                    this.setState({ overrides: { ...overrides } });
+                  }
+                }}
+              />
+            </div>
+          </PopoverSection>
+        </div>
+      );
+    }
+    return ('');
+  }
+
+  renderDisplayConfiguration() {
+    const { color, opacity, style, width } = this.state;
+    const colorScheme = ALL_COLOR_SCHEMES[this.props.colorScheme].slice();
+    if (color && color !== AUTOMATIC_COLOR &&
+      !colorScheme.find(x => x.toLowerCase() === color.toLowerCase())) {
+      colorScheme.push(color);
+    }
+    return (
+      <div>
+        <PopoverSection
+          isSelected
+          onSelect={() => {}}
+          title="Display configuration"
+          info={
+            'Configure your how you overlay is displayed here.'
+          }
+        >
+          <SelectControl
+            name="annotation-layer-stroke"
+            label="Style"
+            // see '../../../../visualizations/nvd3_vis.css'
+            options={[
+              { value: 'solid', label: 'Solid' },
+              { value: 'dashed', label: 'Dashed' },
+              { value: 'wideDashed', label: 'Wide Dashed' },
+              { value: 'dotted', label: 'Dotted' },
+            ]}
+            value={style}
+            onChange={v => this.setState({ style: v })}
+          />
+          <SelectControl
+            name="annotation-layer-opacity"
+            label="Opacity"
+            // see '../../../../visualizations/nvd3_vis.css'
+            options={[
+              { value: '', label: 'Solid' },
+              { value: 'opacityLow', label: '0.2' },
+              { value: 'opacityMedium', label: '0.5' },
+              { value: 'opacityHigh', label: '0.8' },
+            ]}
+            value={opacity}
+            onChange={v => this.setState({ opacity: v })}
+          />
+          <div>
+            <ControlHeader label="Color" />
+            <div style={{ display: 'flex', flexDirection: 'column' }}>
+              <CirclePicker
+                color={color}
+                colors={colorScheme}
+                onChangeComplete={v => this.setState({ color: v.hex })}
+              />
+              <Button
+                style={{ marginTop: '0.5rem', marginBottom: '0.5rem' }}
+                bsStyle={color === AUTOMATIC_COLOR ? 'success' : 'default'}
+                bsSize="xsmall"
+                onClick={() => this.setState({ color: AUTOMATIC_COLOR })}
+              >
+                Automatic Color
+              </Button>
+            </div>
+          </div>
+          <TextControl
+            name="annotation-layer-stroke-width"
+            label="Line Width"
+            isInt
+            placeholder=""
```

Comment:
what about adding `e.g., 1` so they know it's a number vs `1px`?

Reply:
I can get rid of that line alltogether